README.md

```diff
@@ -48,7 +48,7 @@ The current version of the plugin supports next commands:
 
  - **`:CMake`** creates a build directory (if it is necessary) and generates cmake project.
  - **`:CMakeBuild`** builds current cmake project. The command allows to specify cmake target.
- - **`:CTest`** runs tests. The command allow to specify Ctest arguments
+ - **`:CTest`** runs tests. The command allows to specify CTest arguments. Running with a bang ignores g:cmake_ctest_args
```

Comment:
I propose to have 2 items for `:CTest` and `:CTest!` as we already have for `:CMakeRun`